manifests/backend.pp

```diff
@@ -73,13 +111,33 @@
   Boolean $show_diff = true,
   Optional[String] $license_source = undef,
   Optional[String] $license_content = undef,
+  Hash $ad_auths = {},
```

Comment:
are these all of the sensu_ types? is the mapping to just drop `sensu_` from the name of the type?

Reply:
would any of these make sense for the agent instead of the backend?

Reply:
All these are dropping `sensu_` and making plural except silencing is a bit weird.  These all require executing `sensuctl` and thus not supported on agent, this module only supports running `sensuctl` types and providers on backend.  The only custom type that will make sense on agent is `sensu_plugin` and that already supports defining via parameters.